hypervisor/bsp/sbl/platform_acpi_info.c

```diff
@@ -10,13 +10,13 @@ const struct acpi_info host_acpi_info = {
 	6,				/* x86 family: 6 */
 	0x5C,				/* x86 model: 0x5C, ApolloLake */
 	{
-		{SPACE_SYSTEM_IO, 20, 0, 3, 0x400},	/* PM1a EVT */
```

Comment:
acpi_generic_address is using unsigned int format. Clean up all of them

Reply:
Hi Eddie,
This patch is just simplified to make s3 work on SBL.
I have a separate patch which already in mail list to cleanup all misra voilations of this file.